test/gluehbirne_web/views/error_view_test.exs

```diff
@@ -4,11 +4,22 @@ defmodule MindwendelWeb.ErrorViewTest do
   # Bring render/3 and render_to_string/3 for testing custom views
   import Phoenix.View
 
-  test "renders 404.html" do
-    assert render_to_string(MindwendelWeb.ErrorView, "404.html", []) == "Not Found"
+  setup %{conn: conn} do
+    # When rendering the error_page.html.eex the connection is necesary, e.g. because of
+    # <script defer phx-track-static type="text/javascript" src="<%= Routes.static_path(@conn, "/js/app.js") %>"></script>
+    #
+    # Although this MindwendelWeb.ConnCase sets the endpoint, is not avail in the tests. As a fix, I needed to set endpoint explicitely
```

Comment:
Some typos also here in the second part after the comma  ;)

Reply:
Done